package.json

```diff
@@ -16,14 +16,14 @@
     "url": "https://github.com/dcos/dcos-ui"
   },
   "license": "Apache-2.0",
-  "engines": {
-    "node": "4.4.x",
-    "npm": "3.9.x"
-  },
   "browserslist": [
     "last 2 versions",
     "ie 11"
   ],
+  "engines": {
```

Comment:
can we remove this part?

Reply:
I would like to leave it for a while. I know that we have it as a dependency, but as we as a team first need to switch to a node version that supports having these as dependencies and as we most likely will still need to switch to a different node version using `nvm` or something similar we should leave it like that.

Reply:
I'm not talking about removeing the check, just the duplicate part in the package.json.

Reply:
Why duplicate?

Reply:
There's no duplication as far as I can see.

Reply:
The duplication was with the devDependencies, discussed offline that this could be a next PR